### ISSUE TITLE
Crash when accessing archive under chat box

### ISSUE THREAD

**Issue by ariestwn** [zeitlings/ayai-gpt-nexus#9]
Hi,

I'm curious as to why my Alfred crashes when I access the archive through the chat box with ⌃⌘+↵

<img width="1303" alt="SCR-20241020-kxgq" src="https://github.com/user-attachments/assets/da851a9b-c93c-4f12-8383-f44c0a37a743">

**Comment by zeitlings** [comment]
Hey @ariestwn 
Beats me. Did you perhaps double-assign the keyboard shortcut  ⌃⌘+↵?

In Activity Monitor, try to filter the processes by "GPT Nexus" - If you see that a process with that name is still around, your issue might be related to the workflow. If that is the case, force quit the process(es) and try again. 
If there is no such process, the problem might lie elsewhere. 

When did this start happening? Has it always been like that or did something change recently?

**Comment by ariestwn** [comment]
Hi, I found the culprit, it was the Mini Player from Alfred. I don't know why, but they set the mini player binding to ⌃⌘+↵. Maybe it is supposed to run nicely, but because I installed noTunes to block Apple Music from opening, it caused a crash.

So after I clear the hotkey, the problem solved.
<img width="832" alt="image" src="https://github.com/user-attachments/assets/feb308bf-a02b-43bc-8e6a-1a1f99594a5e">
